cmd/up/activate.go

```diff
@@ -441,6 +441,10 @@ func (up *upContext) waitUntilDevelopmentContainerIsRunning(ctx context.Context,
     More information about configuring your persistent volume at https://okteto.com/docs/reference/manifest/#persistentvolume-object-optional`, utils.GetDownCommand(up.Options.ManifestPathFlag)),
 					}
 				}
+				if e.Type == "Warning" && strings.Contains(e.Message, "container veth name provided (eth0) already exists") {
```

Comment:
does the warning disappear at some point?

Reply:
Yes, In this case there's a retry on `kubelet` and it works the second time it retries. I don't know with the rest of warnings, that's why I added the `strings.Contains` to only capture the event for the vets